openmetadata-ui/src/main/resources/ui/src/constants/AdvancedSearch.constants.ts

```diff
@@ -189,13 +189,13 @@ const getCommonQueryBuilderFields = (
 ) => {
   const commonQueryBuilderFields: Fields = {
     deleted: {
-      label: 'Deleted',
+      label: i18next.t('label.deleted'),
```

Comment:
it would be great if we only import the required method instead of importing all methods and using one! in this case, we can import `t` and use at all the places